### PR TITLE
Support configure replicas for each component

### PR DESCRIPTION
**Pull request by clyang82**

### CODE REVIEW

**Comment by metalmatze** [comment]
I'm pretty sure you don't want any of these changes in `jsonnet/vendor`. Please double check what happened there.

**Review by metalmatze — CHANGES_REQUESTED** [review]
Please fix the `jsonnet/vendor` directory changes. Thanks!

**Comment by squat** [comment]
hm btw, if the vendor directory had these weird changes, we should really modify the `build` test so that this would fail. Otherwise, we will almost certainly merge changes like this in the future

File: jsonnet/jsonnetfile.lock.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
           "subdir": ""
         }
       },
-      "version": "73265f5a16de3c1d4b72cdb0f800929c5eb72f03",
-      "sum": "pAo2yL5Ab72oRo6H8gVRkgI3EPqqRDKcEkik+6hhdcA="
+      "version": "0f3097bded073e691bdd799f0edd12d531cc3821",
+      "sum": "0pZPPJCisYuXsVppWe7mJK1e9N6qxMzDDC7uJuAel7M="
```
Comment (clyang82) [inline jsonnet/jsonnetfile.lock.json:63]: @metalmatze I need to have `jsonnet/vendor` directory changes to have this PR https://github.com/observatorium/deployments/pull/339 in. please see the changes ^^^

**Comment by metalmatze** [comment]
This was really confusing, but the actual deletion in vendor folders is coming from `jsonnet/vendor/github.com/observatorium/deployments/operator/jsonnet/vendor/`. As we removed that folder in the observatorium/deployments repository this is fully expected then and exactly what we want.

**Review by metalmatze — APPROVED** [review]

**Pull request merged**